client/src/App.tsx

```diff
@@ -1,14 +1,12 @@
 import { ReactElement } from 'react';
 import { ThemeProvider } from '@emotion/react';
-import RootLayout from './layouts/RootLayout';
 import LightTheme from './themes';
-import {Header} from './components';
+import LoginWrapper from './components/auth/LoginWrapper';
 
 const App = (): ReactElement => {
   return (
     <ThemeProvider theme={LightTheme}>
-      {/* <RootLayout /> */}
-      <Header key={null} type={''} props={undefined} />
+      <LoginWrapper isPlayer isSignup />
```

Comment:
we can handle the forms by the location Without the need to use props everywhere
using useLocation you can know the endpoint of your page 
like this example : 
you can search about it to know more ...